daffodil-runtime1/src/main/scala/edu/illinois/ncsa/daffodil/processors/parsers/PackedBinaryTraits.scala

```diff
@@ -101,7 +100,7 @@ abstract class PackedBinaryIntegerBaseParser(
       val int = toBigInteger(dis.getByteArray(nBits, start))
       start.simpleElement.overwriteDataValue(int)
```

Comment:
It looks like the ``signed`` variable in PackedBinaryIntegerBaseParser is never used. I think you just need to throw a PE if signed is false but the ``val int`` result of toBigInteger is negative? Also, while you're in this section, it doesn't look like the ``signed`` variable needs a default value. All uses of this base class pass in the signed variable.

+1 with those two minor issues fixed.

Reply:
BCD relies on using the default value.  PackedBinaryIntegerBaseParser is the base parser for all three packed formats integer parsers.  I know the naming I used here might be a bit confusing, but the spec also uses "packed" to refer to packed decimal, bcd, and ibm4690Packed formats.  I figured PackedBinary was a good compromise for something that all 3 formats use.

Reply:
Ah yep, you're right. I missed that BCD, the default is fine.